yt_dlp/extractor/bilibili.py

```diff
@@ -892,19 +892,28 @@ def _get_subtitles(self, *, ep_id=None, aid=None):
                 'aid': aid,
             })) or {}
         subtitles = {}
-        for sub in sub_json.get('subtitles') or []:
-            sub_url = sub.get('url')
-            if not sub_url:
-                continue
-            sub_data = self._download_json(
-                sub_url, ep_id or aid, errnote='Unable to download subtitles', fatal=False,
-                note='Downloading subtitles%s' % f' for {sub["lang"]}' if sub.get('lang') else '')
-            if not sub_data:
-                continue
-            subtitles.setdefault(sub.get('lang_key', 'en'), []).append({
-                'ext': 'srt',
-                'data': self.json2srt(sub_data)
-            })
+
+        for sub in sub_json.get('video_subtitle') or []:
+            srt_sub = sub.get('srt')
+            ass_sub = sub.get('ass')
+            sub_formats = subtitles.setdefault(sub.get('lang_key', 'en'), [])
+
+            if srt_sub:
+                sub_data = self._download_json(
+                    srt_sub.get('url'), ep_id or aid, errnote='Unable to download subtitles', fatal=False,
+                    note='Downloading subtitles%s' % f' for {sub["lang"]} ({sub["lang_key"]})' if sub.get('lang') else '')
```

Comment:
Same with this, the `lang_key` might not be present. Maybe save the `lang_key` above into a variable and reuse here?

Reply:
This is done.